packages/teams-js/test/public/nestedAppAuth.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { errorLibraryNotInitialized } from '../../src/internal/constants';
+import { app, FrameContexts, nestedAppAuth } from '../../src/public';
+import { _minRuntimeConfigToUninitialize, Runtime } from '../../src/public/runtime';
+import { Utils } from '../utils';
+
+/* eslint-disable */
+/* As part of enabling eslint on test files, we need to disable eslint checking on the specific files with
+   large numbers of errors. Then, over time, we can fix the errors and reenable eslint on a per file basis. */
+
+/**
+ * Test cases for nested app auth APIs
+ */
+describe('nestedAppAuth', () => {
```

Comment:
Can you verify if we need framed or frameless options for UTs here?

Reply:
Since we are just setting  runtimeConfig in the tests and not sending any messages, I think this shouldn't be needed.

Reply:
Thanks for confirming @jadahiya-MSFT. As discussed yesterday @shrshindeMSFT, we should be good to close this conversation. Feel free to open it if you have any further concerns.